flashcards/backend/comments/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+
+from comments.models import Comment
+
+
+class CommentAdminConfig(admin.ModelAdmin):
+    """Class **CommentAdminConfig** displays comments list in admin panel
+    based on :class:`comments.models.Comment` model."""
+
+    list_display = (
+        "user",
+        "flashcard",
+        "deck",
+        "description",
```

Comment:
is this comment it self or a description of a comment? I think it is a bit misleading.